mithril-aggregator/src/database/record/signed_entity.rs

```diff
@@ -259,6 +259,27 @@ impl TryFrom<SignedEntityRecord> for CardanoStakeDistributionMessage {
     }
 }
 
+impl TryFrom<SignedEntityRecord> for CardanoStakeDistributionListItemMessage {
+    type Error = StdError;
+
+    fn try_from(value: SignedEntityRecord) -> Result<Self, Self::Error> {
+        #[derive(Deserialize)]
+        struct TmpCardanoStakeDistribution {
+            epoch: Epoch,
+            hash: String,
+        }
+        let artifact = serde_json::from_str::<TmpCardanoStakeDistribution>(&value.artifact)?;
+        let message = CardanoStakeDistributionListItemMessage {
+            epoch: artifact.epoch,
```

Comment:
Same comment for the epoch used.

Reply:
Fixed in this commit: https://github.com/input-output-hk/mithril/commit/3b440ded30f374e0d3d74603047961aa919c92ab